src/components/FormView.jsx

```diff
@@ -55,13 +63,10 @@ const FormView = ({
     ? 'GoogleReCaptcha'
     : null;
 
-  let validToken = useRef('');
-  const onVerifyCaptcha = useCallback(
-    (token) => {
-      validToken.current = token;
-    },
-    [validToken],
-  );
+  let [validToken, setValidToken] = React.useState();
+  const onVerifyCaptcha = useCallback((token) => {
+    setValidToken(token);
+  }, []);
```

Comment:
why useEffect triggers now on component load and not on changing validToken? 
tokenk could be became valid after the component load..